packages/service-clients/azure-client/src/AzureClient.ts

```diff
@@ -59,6 +67,32 @@ const azureClientFeatureGates = {
 	"Fluid.Container.ForceWriteConnection": true,
```

Comment:
@andre4i - This will be required in perpetuity, right?  i.e. confirming that it doesn't belong in `azureClientV1CompatFeatureGates`

Reply:
yes.